### DISCUSSION TITLE
Elasticsearch shards closing

### DISCUSSION THREAD

**Discussion by its-a-me-mario-the-og** [Security-Onion-Solutions/securityonion#8719] [Unsupported Versions]
Hi, my elasticsearch service keeps showing error and elastalert is showing as missing. These are the two only services that I am having trouble with. I am on the latest SO 2.3.160, it is a standalone node with a forward node connected to it. It has 16 GB RAM, along with 4 core cpu, and 5 TB of SSD. I am adding what the elasticsearch log is showing, and there are no logs for elastalert. I have tried several different troubleshooting techniques but nothing has worked so far. Here is the elasticsearch logs down below. Any advice or help will be very helpful, and thank you in advance for your help!

[elasticsearch log](https://github.com/Security-Onion-Solutions/securityonion/files/9548704/elasticsearch.log)

**Comment by dougburks** [comment]
Did you update from an earlier version? If so, did you follow all steps at https://docs.securityonion.net/en/2.3/soup.html#elastic-8?

**Reply by its-a-me-mario-the-og** [reply]
Yes sir it was bein updated from 2.3.130, but I did not make the changes before going up to 2.3.160, what do I need to do in order to get the it back up and running?

**Reply by dougburks** [reply]
From your elasticsearch.log:
> java.lang.IllegalStateException: Failed to parse mappings for index [[elastalert_status_error/x0mywviZRG2yvhVP2g1V5w]]

You may need to manually delete your elastalert indices. Alternatively, once [you've upgraded to 32GB RAM as we talked about in your other discussion](https://github.com/Security-Onion-Solutions/securityonion/discussions/8699), you may just want to wipe and reinstall.

**Reply by its-a-me-mario-the-og** [reply]
This is a different setup from the other conversation, but once I manually delete the indices, is there a command that should rebuild them? Or do I need to try ./so-setup-network and reconstruct the cluster?

**Reply by dougburks** [reply]
When you restart elastalert, it should try to create new indices. If all else fails, you can re-rerun Setup or wipe and reinstall.

**Reply by its-a-me-mario-the-og** [reply]
If I need to re-run Setup, do I need to get rid of the network settings already on the box? If so where would I need to go?

**Reply by dougburks** [reply]
You should be able to use the same network settings.

**Comment by dlee35** [comment]
From a quick glance, I saw this referenced in your attached log:

"org.elasticsearch.index.mapper.MapperParsingException: Failed to parse
mapping: Error parsing [format] on field ***@***.***: Invalid format:
[dateOptionalTime]: Unknown pattern letter: t"

It appears that either one of the logs you're ingesting isn't appropriately
mapping timestamp or the parser is potentially attempting to store an
improperly formatted value. Is there a timestamp formatting discrepancy in
what the parser is expecting and what the incoming log has defined?


Thanks,

Dustin
